<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentWidget</class>
 <widget class="QWidget" name="DepartmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1342</width>
    <height>762</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>77</red>
        <green>77</green>
        <blue>77</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>212</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>77</red>
        <green>77</green>
        <blue>77</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>212</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="descriptionText">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="myVolunteersList">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="unassignButton">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Unassign selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="unassignedList">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="assignButton">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Assign selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <widget class="QPushButton" name="showMostSuitable">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Show only the most suited</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
